components/blitz/resources/omero/api/ITimeline.ice

```diff
@@ -22,55 +22,59 @@ module omero {
         /**
          * Service for the querying of OMERO metadata based on creation and
          * modification time. Currently supported types for querying include:
+         * <ul>
+         *   <li>{@link omero.model.Annotation}</li>
+         *   <li>{@link omero.model.Dataset}</li>
+         *   <li>{@link omero.model.Image}</li>
+         *   <li>{@link omero.model.Project}</li>
+         *   <li>{@link omero.model.RenderingDef}</li>
+         * </ul>
          *
-         *    - {@link omero.model.Annotation}
-         *    - {@link omero.model.Dataset}
-         *    - {@link omero.model.Image}
-         *    - {@link omero.model.Project}
-         *    - {@link omero.model.RenderingDef}
+         * <h4>Return maps</h4>
          *
-         * Return maps:
-         * -----------
          * The map return values will be indexed by the short type name above:
-         * <i>Project</i>, <i>Image</i>, ... All keys which are passed in the StringSet
-         * argument will be included in the returned map, even if they have no
-         * values. A default value of 0 or the empty list \[] will be used.
+         * <i>Project</i>, <i>Image</i>, ... All keys which are passed in the
```

Comment:
The ellipsis should probably be changed, '...' -> 'etc.' while you are fixing this doc.